pkg/apis/machineconfiguration.openshift.io/v1/helpers.go

```diff
@@ -29,6 +30,23 @@ func MergeMachineConfigs(configs []*MachineConfig, osImageURL string) *MachineCo
 		}
 		outIgn = ign.Append(outIgn, configs[idx].Spec.Config)
 	}
+
+	// sets the KernelType if specified in any of the MachineConfig
+	for _, cfg := range configs {
+		if cfg.Spec.KernelType == "realtime" {
+			kernelType = cfg.Spec.KernelType
+			break
+		}
+		if kernelType == "default" && cfg.Spec.KernelType != "realtime" {
```

Comment:
nit: Could be a shorter `else if` but this should be fine as is.

Reply:
done